src/main/java/org/mockito/internal/junit/JUnitRule.java

```diff
@@ -14,25 +13,74 @@
 public class JUnitRule implements MockitoRule {
 	
 	private final MockitoLogger logger;
+    private final boolean silent;
 
-	public JUnitRule(MockitoLogger logger) {
+    public JUnitRule(MockitoLogger logger, boolean silent) {
```

Comment:
Maybe use an enum instead of a boolean, the benefit would allow self documenting.